v2/components/UI/card.jsx

```diff
@@ -0,0 +1,54 @@
+import PropTypes from 'prop-types';
+import { Heading } from '.';
+
+const Card = ({ image, title, date, description, onClick }) => (
+  <div className="card" onClick={onClick}>
+    <img src={image} />
+    <div className="card-header">
+      <Heading title={title} />
+    </div>
+    <div className="card-body">
+      <span className="date">{date}</span>
+      <p>{description}</p>
+    </div>
+    <style jsx>
+      {`
+        .card {
+          display: flex;
+          flex-direction: column;
+          border-radius: 1rem;
+          margin: 1rem;
+          box-shadow: 0 0.5rem 1rem 2px rgba(0, 0, 0, 0.1);
```

Comment:
Mixing pixel and rem values here
```suggestion
          box-shadow: 0 0.5rem 1rem 0.2rem rgba(0, 0, 0, 0.1);
```